stattutor/stattutor.py

```diff
@@ -0,0 +1,386 @@
+"""
+This is a XBlock used to serve the CTAT based StatTutor problems originally
+implemented for OLI (http://oli.cmu.edu/).
+"""
+
+import re
+import uuid
+import base64
+import math
+import pkg_resources
+import bleach
+
+# pylint: disable=import-error
+# The xblock packages are available in the runtime environment.
+from xblock.core import XBlock
+from xblock.fields import Scope, Integer, String, Float, Boolean
+from xblock.fragment import Fragment
+# pylint: enable=import-error
+
+
+class StattutorXBlock(XBlock):
+    """
+    A XBlock providing a CTAT backed StatTutor.
+    """
+    # pylint: disable=too-many-instance-attributes
+    # All of the instance variables are required.
+
+    display_name = String(
+        help="Display name of the component",
+        default="StatTutor",
+        scope=Scope.content)  # required to prevent garbage name at the top
+
+    # **** xBlock tag variables ****
+    # The width must be at least 900 in order to accommodate some dynamically
+    # loaded images and some of the interactive elements without causing
+    # side scrolling scrollbars to appear.  They are set here instead of
+    # hard coding them into ctatxblock.html to make it easier for EdX
+    # administrators to modify them if they wish without having to scour
```

Comment:
That was actually my point from my original request [1]...
If it only looks right with one set of dimensions,
we _don't_ want to change it, ever.
So let's just hard-code it directly in the HTML template.

I also suggested that it may fit in 800px, which as you point out, has edge-case issues,
so I have no objection to keeping it at 900px.

[1] https://github.com/CMUCTAT/XBlockStattutor/pull/21